src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -73,7 +73,7 @@ internal static DerSequenceReader CreateForPayload(byte[] payload)
         {
             return new DerSequenceReader(true, payload);
         }
-
+        
```

Comment:
Running `git diff --check` before staging the files should help you find these. It highlights them in console-red.